OrbitQt/orbitmainwindow.cpp

```diff
@@ -257,15 +257,15 @@ bool OrbitMainWindow::HideTab(QTabWidget* a_TabWidget, const char* a_TabName) {
 }
 
 //-----------------------------------------------------------------------------
-std::wstring OrbitMainWindow::FindFile(const std::wstring& a_Caption,
-                                       const std::wstring& a_Dir,
-                                       const std::wstring& a_Filter) {
+std::string OrbitMainWindow::FindFile(const std::string& caption,
+                                      const std::string& dir,
+                                      const std::string& filter) {
   QStringList list = QFileDialog::getOpenFileNames(
-      this, ws2s(a_Caption).c_str(), ws2s(a_Dir).c_str(),
-      ws2s(a_Filter).c_str());
-  std::wstring result;
+      this, caption.c_str(), dir.c_str(),
```

Comment:
clang-format I think would put them all on the same line.

Reply:
done - clang-formated the commit.